redis/tests/test_redis.py

```diff
@@ -1,23 +1,24 @@
 import time
+from testcontainers.core.waiting_utils import wait_for_logs
 
 from testcontainers.redis import RedisContainer
 
 
 def test_docker_run_redis():
-    config = RedisContainer()
-    with config as redis:
+    with RedisContainer('redis:7.0.11') as redis:
+        wait_for_logs(redis, 'Ready to accept connections')
         client = redis.get_client()
-        p = client.pubsub()
-        p.subscribe('test')
+        pubsub = client.pubsub()
+        pubsub.subscribe('test')
         client.publish('test', 'new_msg')
-        msg = wait_for_message(p)
+        msg = wait_for_message(pubsub)
         assert 'data' in msg
         assert b'new_msg', msg['data']
 
 
 def test_docker_run_redis_with_password():
-    config = RedisContainer(password="mypass")
-    with config as redis:
+    with RedisContainer('redis:7.0.11', password="mypass") as redis:
+        wait_for_logs(redis, 'Ready to accept connections')
```

Comment:
Same, should we move this into the container implementation?

Reply:
> Regarding keycloack, there's a possible alternative image discussed in #289, but we don't have a PR yet.
I tried for a few minutes to get the maintained image up and running, but at first sight it looked like it is intended that you specify at least an `ENTRYPOINT` while using the image as a base image. E.g. it is not 100% Plug'n'Play. I might be wrong though, I did not give myself too much time trying to fix it. 

I might have some time to look into it at a later stage, but I really want to get `.with_reuse()` up and running after the Ryuk PR is merged. Might be that it annoys me enough during development that I cannot get all the tests to run on my machine 😅